packages/remix-dev/codemod/replace-remix-magic-imports/transform.ts

```diff
@@ -0,0 +1,229 @@
+import { type NodePath } from "@babel/core";
+import * as t from "@babel/types";
+import _ from "lodash";
+
+import createTransform2 from "../createTransform";
```

Comment:
This can just be called `createTransform` I guess?

```suggestion
import createTransform from "../createTransform";
```